templates/config/prometheus.yml.tpl

```diff
@@ -376,3 +376,14 @@ scrape_configs:
       - '{{.}}'
     {{- end}}
 {{- end}}
+
+# WARNING: This file is auto-generated. Do not edit! All your modification will be overwritten!
+# You can use 'tiup cluster edit-config' and 'tiup cluster reload' to update the configuration
+# All configuration items you want to change can be added to:
+# server_configs:
+#   prometheus:
+#     aa.b1.c3: value
+#     aa.b2.c4: value
+{{- if .RemoteConfig}}
+{{.RemoteConfig}}
```

Comment:
What if the user try to define `rule_files` and `scrape_configs` in Prometheus.Config?

Reply:
I prefer to remove `global`, `rule_files`, `alerting`, `scrape_configs` from the config the user input.

Reply:
ditto

Reply:
fixed by unmarshal and then marshal